demos/appyx-navigation/web/src/jsMain/kotlin/com/bumble/appyx/navigation/Main.kt

```diff
@@ -123,6 +125,27 @@ private fun BlackContainer(content: @Composable () -> Unit) {
     }
 }
 
+@Composable
+private fun ProvideScopeConfiguration(
+    navigator: Navigator,
+    screenSize: ScreenSize,
+    body: @Composable () -> Unit
+) {
+    CompositionLocalProvider(
+        LocalDensity provides screenSize.calculateDensityFromScreenSize(),
+        LocalNavigator provides navigator,
+    ) {
+        body()
+    }
+}
+
+private fun ScreenSize.calculateDensityFromScreenSize(): Density =
+    if (widthDp.value < 1500) {
+        Density(1.2f, 1f)
+    } else {
+        Density(1.8f, 1.2f)
+    }
```

Comment:
This looks weird to me. Why is it specifically `1500`? Why specifically `1.2f`, `1f`, `1.8f`, `1.2f`? The issue title also implies dynamic, but this looks more two static cases rather?